modules/react-components/src/layouts/page.tsx

```diff
@@ -40,6 +41,10 @@ export interface PageLayoutPropsInterface extends PageHeaderPropsInterface, Test
      * Flag to enable/disable padding.
      */
     padded?: boolean;
+    /**
+     * page name to display in the browser tab
```

Comment:
```suggestion
     * Page name to display in the browser tab.
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3438